Quadriga/src/main/java/edu/asu/spring/quadriga/service/workbench/mapper/impl/ProjectShallowMapper.java

```diff
@@ -240,4 +241,36 @@ public IProject getProjectDetails(ProjectDTO projectDTO) throws QuadrigaStorageE
 		return projectList;
 	}
 	
+	/**
+	 * {@inheritDoc}
+	*/
+	@Override
+	@Transactional
+	public List<IProject> getProjectListByAccesibility(String accessibility)
+			throws QuadrigaStorageException{
+		List<ProjectDTO> projectDTOList = dbConnect.getAllProjectsDTOByAccessibility(accessibility);
+		List<IProject> projectList = null;
+		if(projectDTOList!=null){
+			for(ProjectDTO projectDTO : projectDTOList){
+				if(projectList == null){
+					projectList = new ArrayList<IProject>();
+				}
+				IProject projectProxy = new ProjectProxy(projectManager);
+				projectProxy.setProjectId(projectDTO.getProjectid());
+				projectProxy.setProjectName(projectDTO.getProjectname());
+				projectProxy.setDescription(projectDTO.getDescription());
+				projectProxy.setProjectAccess(EProjectAccessibility.valueOf(projectDTO.getAccessibility()));
+				projectProxy.setUnixName(projectDTO.getUnixname());
+				projectProxy.setOwner(userDeepMapper.getUser(projectDTO.getProjectowner().getUsername()));
+				projectProxy.setCreatedBy(projectDTO.getCreatedby());
+				projectProxy.setCreatedDate(projectDTO.getCreateddate());
+				projectProxy.setUpdatedBy(projectDTO.getUpdatedby());
+				projectProxy.setUpdatedDate(projectDTO.getUpdateddate());
+				projectList.add(projectProxy);
+			}
+		}
+		
+		return projectList;
+	}
+	

```

Comment:
remove this and see my previous comment:
"Let your manager class get the DTOs and then map them using the mapper. Then you can get rid of the getProjectListByAccesibility method in the mapper classes."
You mapper only needs to map one object at a time (there should already be the method for that).